fun.kikoplay.KikoPlay.yml

```
@@ -0,0 +1,243 @@
+app-id: fun.kikoplay.KikoPlay
+branch: stable
+runtime: org.kde.Platform
+runtime-version: '5.15-23.08'
+sdk: org.kde.Sdk
+command: KikoPlay
+finish-args:
+  - --device=dri
+  - --share=ipc
+  - --share=network
+  # aria2c download
+  - --filesystem=xdg-videos
+  - --filesystem=xdg-download
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  # for hardware acceleration
+  - --device=dri
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --persist=.config/kikoplay
+
+add-extensions: 
+  org.freedesktop.Platform.ffmpeg-full:
+    directory: lib/ffmpeg
+    version: 23.08
+    add-ld-path: .
+cleanup-commands:
+  - mkdir -p ${FLATPAK_DEST}/lib/ffmpeg
+
+modules:
+  - shared-modules/luajit/luajit.json
+
+  - name: libass
+    cleanup:
+      - /include
+      - /lib/*.la
```

```suggestion
      - "*.la"
```

you also probably need "*.a"